packages/config/src/bridges/sygma.ts

```diff
@@ -0,0 +1,282 @@
+    detailedDescription: 
+      `Sygma seeks to solve interoperability challenges via innovations in dynamic cross-chain message routing and Zero-knowledge (Spectre) and Optimistic (Zipline) co-processing capabilities. The system is designed to automatically take a user's context into account, selecting the most optimal combination of verification systems and tailoring its security path, abstracting even the most arduous cross-chain complexities.`,
+    links: {
+      websites: [
+        'https://buildwithsygma.com/',
+        'https://blog.buildwithsygma.com',
+      ],
+      documentation: [
+        `https://docs.buildwithsygma.com`,
+      ],
+      explorers: [
+        `https://scan.buildwithsygma.com`,
+        `https://scan.test.buildwithsygma.com`
+      ],
+      repositories: ['https://github.com/sygmaprotocol'],
+      socialMedia: [
+        'https://discord.gg/Qdf6GyNB5J',
+        'https://twitter.com/buildwithsygma',
+        `https://t.me/buildwithsygma`
+      ],
+      apps: [
+        `https://subbridge.io`,
+        'https://transfer-ui.test.buildwithsygma.com/transfer',
+        'https://faucet-ui-stage.buildwithsygma.com/',
+        'https://validator.faucet.chainsafe.dev/upload',
+      ]
+    },
+  },
+  riskView: {
+    validatedBy: {
+      value: 'Third party',
+      description:
+        '4/6 of the MPC group (the Sygma relayer network) is required to create a cross-chain message with the MPC signature.', // sygma eng team currently developing a zk methodology for block header oracle verification which we will implement on mainnet relayers soon. additionally, optimistic approach coming as well. we call this "tailored security"
+      sentiment: 'neutral',
+    },
+    sourceUpgradeability: {
+      value: 'No',
+      description: 'Contracts are not upgradable.',
+      sentiment: 'good',
+    },
+    destinationToken: RISK_VIEW.CANONICAL_OR_WRAPPED,
+  },
+  technology: {
+    destination: [ // these are the currently supported networks on mainnet, but the main integration available is our backend integration for PHA tokens between the EVM<->Phala/Khala routes on Phala's SubBridge 
+      'Ethereum',
+      'Phala',
+      'Khala',
+      'Polygon',
+      'Cronos',
+      'Base',
+      'Gnosis',
+    ],
+    principleOfOperation: {
+      name: 'Principle of operation',
+      description:
+        `Sygma uses verification systems that have been architected to provide the most optionality in order to reach cross-chain consensus. The tailored security verification systems using multi-party computation (MPC), Spectre (ZK), and Zipline (Optimistic) balances degrees of trust minimization and can be combined to ensure the highest level of security.`,
```

Comment:
Zipline and Spectre are not yet in production, not sure if we here are suggesting too much. Thoughts?

Reply:
the MPC verification system _is_ in production. however, the tailored security system is a foundational feature of our protocol so it is still worth mentioning even if those components are still only in testnet. is this still viable @lucadonnoh ?

Reply:
no, we don’t want to mention things that are not on production. once they are they will be mentioned

Reply:
fair fair. updated again :)